applications/AdjointFluidApplication/custom_utilities/numerical_diffusion.h

```diff
@@ -0,0 +1,377 @@
+//  KratosAdjointFluidApplication
+//
+//  License:		 BSD License
+//					 license: AdjointFluidApplication/license.txt
+//
+//  Main authors:    Suneth Warnakulasriya, https://github.com/sunethwarna
+//
+
+#if !defined(KRATOS_NUMERICAL_DIFFUSION)
+#define KRATOS_NUMERICAL_DIFFUSION
+
+// System includes
+#include <vector>
+#include <string>
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/process_info.h"
+#include "includes/model_part.h"
+#include "includes/ublas_interface.h"
```

Comment:
you should not need to include explicitly "ublas_interface" since it is included in define.h

please don't include it since it will break while porting to AMatrix